### ISSUE TITLE
Invalid or corrupt jarfile /home/<username>/.conscript/sbt-launch.jar

### ISSUE THREAD

**Issue by mergenchik** [foundweekends/conscript#29]
Hi, this is my first issue posting on `github`. So, excuse me if something goes wrong :)

I tried to install `giter8`, installation of `conscript` were without any errors, by using `curl` script. But running `cs` with

```
cs n8han/giter
```

gave me errror

```
Error: Invalid or corrupt jarfile /home/mergenchik/.conscript/sbt-launch.jar
```

I don't know why it happened, but replacing `sbt-launch.jar` with mine fixed the error.  

I'd like to post it somewhere so anybody who catches same error will known what to do.

**Comment by ShardPhoenix** [comment]
I had SBT 0.13.8 installed, but CS came with 0.13.7. I had to overwrite the 0.13.7 version with the 0.13.8 version (from my SBT install) to get it to work.

**Comment by nickdb93** [comment]
I'm having a similar issue, but the softlink is pointing to the correct version. Any suggestions?

**Comment by trepidacious** [comment]
I think the link (sbt-launch.jar) is a link as expected, but the target (sbt-launch-0.13.7.jar) is an error page as described - maybe the URL used to get sbt is no longer valid?

In any case I removed sbt-launch.jar and sbt-launch-0.13.7.jar, and created a new symbolic link from ~/.conscript/sbt-launch.jar to the jar I already had in my normal SBT install. This seems to work fine.

Is it worth reopening this? It still seems to be broken...

**Comment by trepidacious** [comment]
Yup, it's trying to get the file from:

http://typesafe.artifactoryonline.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/0.13.7/sbt-launch.jar

... which is a 404, see issue #73

**Comment by nickdb93** [comment]
Where is the jar included with SBT located?

**Comment by thobson** [comment]
@nickdb93 I would suggest using " $ which sbt " to find your sbt install, then look in $SBT_HOME/libexec for the sbt-launch.jar, finally symlink it i.e. $ cd .conscript; rm sbt-launch*; ln -s $SBT_HOME/libexec/sbt-launch.jar sbt-launch.jar

**Comment by jcranky** [comment]
It is still broken, just installed from scratch, following the instructions, and got this error. This is quite confusing and I also think this issue should be re-opened.

**Comment by eed3si9n** [comment]
@jcranky I think the underlying issue is #73

**Comment by jcranky** [comment]
@eed3si9n  I see - should be fixed now then, cool :+1:

**Comment by nickdb93** [comment]
Confirmed this is fixed. Thank you all

**Comment by prateekpg2455** [comment]
Hi,

I have been trying to install spark on my MacOSX. I have already installed Scala and Sbt using brew. After downloading and untarring the spark-1.4.1 (i have also tried spark-1.2.1 and spark-1.1.1) I am running - 
sbt/sbt clean assembly

Everytime its the same error: 
Invalid or corrupt jarfile sbt/sbt-launch-0.13.5.jar

I have tried above mentioned solutions of changing the name and manually installing sbt, but none of those seems to work. I have seen people asking questions on stackoverflow lot of times and solution does not seem to be consistent. Can some one help me here?
